src/components/molecules/TableHeader/TableHeader.tsx

```diff
@@ -149,62 +134,59 @@ const TableHeader: React.FC<TableHeaderProps> = ({
     };
   }, [leaveSeat]);
 
-  const updateTableTitle = useCallback(
-    async (newTableTitle) => {
+  const [updateResponse, updateTables] = useAsyncFn(
+    async (values: EditTableForm) => await updateTableInfo(values),
+    []
+  );
+
+  const updateTableInfo = useCallback(
```

Comment:
Since this is only called above in `useAsyncFn`, would it be better to just define it inline there?

Reply:
I believe so.

Reply:
This has been fixed in: https://github.com/sparkletown/sparkle/pull/1377/commits/4b7532e6f02068126c212a6f0dab1714aaaad55a

Reply:
The whole logic can be moved in `onSubmit` but I've decided to use `useAsyncFn` for better error handling and keep it as a `const` I've also added a note.

Reply:
Basically the same question as earlier, but what is the point of `onSubmit` now? It just seems to be a wrapper around `updateTables` (defined above). Can we just pass that directly into the `onSubmit` prop and get rid of the `onSubmit` `const`/`useCallback` altogether?

Reply:
Yes, it's fixed in: https://github.com/sparkletown/sparkle/pull/1377/commits/c222264f9875e83b03a9a64b49c9ae1d57e9401c